win32ss/reactx/ntddraw/dvp.c

```diff
@@ -190,15 +187,14 @@ NtGdiDvpGetVideoPortLine(HANDLE hVideoPort,
 
     if (pfnDvpGetVideoPortLine == NULL)
     {
-		DPRINT1("Warning: no pfnDvpGetVideoPortLine");
+		DPRINT1("Warning: no pfnDvpGetVideoPortLine\n");
```

Comment:
We seem to have indentation with tabs here, can this be fixed?

Reply:
Good catch. Fixed.